ext/pcre/php_pcre.c

```diff
@@ -853,6 +870,17 @@ PHPAPI pcre_cache_entry* pcre_get_compiled_regex_cache_ex(zend_string *regex, bo
 		return NULL;
 	}
 
+	/* Compute and cache the subpattern table to avoid computing it again over and over. */
+	new_entry.subpats_table = make_subpats_table(name_count, &new_entry, !PCRE_G(per_request_cache));
```

Comment:
Previously, the table was computed only when named capture groups were present.

Reply:
Yes, it's a draft after all. In particular, this needs a check that name_count>0.